dlrs/main/lwc/rollupEditor/rollupEditor.html

```diff
@@ -1,269 +1,542 @@
+              </button>
+            </h3>
+
+            <div  class="slds-section__content slds-grid slds-wrap slds-gutters">
+
+                <div class="slds-col slds-size_1-of-1 slds-medium-size_1-of-2">
+                  <!-- Field to Aggregate	 -->
+                  <div class="slds-m-top_x-small">
+                    <lightning-input
+                      type="text"
+                      name="rollup_FieldToAggregate__c"
+                      data-name="rollup_FieldToAggregate__c"
+                      label="Field to Aggregate"
+                      value={rollup.FieldToAggregate__c}
+                      required
+                    ></lightning-input>
+                    <c-rollup-editor-error
+                      errors={errors.FieldToAggregate__c}
+                    ></c-rollup-editor-error>
+                  </div>
+
+                  <!-- Field to Order By	 -->
+                  <div class="slds-m-top_x-small">
+                    <lightning-input
+                      type="text"
+                      name="rollup_FieldToOrderBy__c"
+                      data-name="rollup_FieldToOrderBy__c"
+                      label="Field to Order By"
+                      value={rollup.FieldToOrderBy__c}
+                    ></lightning-input>
+                    <c-rollup-editor-error
+                      errors={errors.FieldToOrderBy__c}
+                    ></c-rollup-editor-error>
+                  </div>
+
+                  <!-- Aggregate Options -->
+                  <div class="slds-m-top_x-small">
+                    <lightning-combobox
+                      name="rollup_AggregateOperation__c"
+                      data-name="rollup_AggregateOperation__c"
+                      label="Aggregate Operation"
+                      placeholder="Select"
+                      options={aggregateOptions}
+                      value={rollup.AggregateOperation__c}
+                      required
+                    ></lightning-combobox>
+                    <c-rollup-editor-error
+                      errors={errors.FieldToOrderBy__c}
+                    ></c-rollup-editor-error>
+                  </div>
+
+                  <!-- Aggregate Result Field -->
+                  <div class="slds-m-top_x-small">
+                    <lightning-input
+                      type="text"
+                      name="rollup_AggregateResultField__c"
+                      data-name="rollup_AggregateResultField__c"
+                      label="Aggregate Result Field"
+                      value={rollup.AggregateResultField__c}
+                    ></lightning-input>
+                    <c-rollup-editor-error
+                      errors={errors.AggregateResultField__c}
+                    ></c-rollup-editor-error>
+                  </div>
+
+                  <!-- Aggregate All Rows -->
+                  <div class="slds-m-top_x-small">
+                    <lightning-input
```

Comment:
Do we want to make all Checkboxes Toggles? (I realize I've already requested the Active toggle be removed)

I could go either way on standardizing on Checkbox vs. toggle.

Reply:
We should keep standardizing on Checkbox